torch_geometric/distributed/dist_context.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+from typing import Optional
+
+
+class DistRole(Enum):
+    WORKER = 1  # for worker mode
```

Comment:
this will give the extension for different distributed mode. Worker mode is just one of them and maybe later we can have server/client distributed mode and sampler /trainer are in different nodes, etc.

Reply:
Can you still add a bit of documentation here?